features/user_sign_up.feature

```diff
@@ -0,0 +1,26 @@
+Feature: User sign-up
+
+    As a user
+    In order to use this messaging service
+    would like to to sign-up
```

Comment:
`I would like to sign-up`

Reply:
wait what, i did change that before i submitted it wtf??